### ISSUE TITLE
[Bug]: van_hove analysis 3d plot function doesnt work!

### ISSUE THREAD

**Issue by okibrisli** [materialsvirtuallab/pymatgen-analysis-diffusion#396]
### Email (Optional)

orhankibrisli@gmail.com

### Version

v2024.6.25.1

### Which OS(es) are you using?

- [ ] MacOS
- [ ] Windows
- [X] Linux

### What happened?

@shyuep similar problems persist in this one too. 1d plotting works fine tho!

### Code snippet

```python
for i, analyzer in enumerate(diff_analyzer):
    try:
        print(f"Starting Van Hove Analysis for analyzer {i} at {temperatures[i]}K")

        van_hove = VanHoveAnalysis(
            analyzer,
            avg_nsteps=avg_nsteps,
            ngrid=ngrid,
            rmax=rmax,
            step_skip=step_skip,
            sigma=sigma,
            cell_range=cell_range,
            species=diffusing_species,
            reference_species=reference_species,
            indices=indices
        )

        # Save 3d plot
        van_hove_3d_plot = van_hove.get_3d_plot(mode='distinct')
        van_hove_3d_plot_filename = os.path.join(output_dir, f'van_hove_3d_plot_{i}.png')
        plt.savefig(van_hove_3d_plot_filename)
        plt.close()  # Close the figure to free memory
        print(f"3D plot saved for analyzer {i} at {temperatures[i]}K to {van_hove_3d_plot_filename}")
```

### Log output

```shell
Starting Van Hove Analysis for analyzer 0 at 800K
Unexpected error for analyzer 0 at 800K: VanHoveAnalysis.get_3d_plot() got an unexpected keyword argument 'ax'
Starting Van Hove Analysis for analyzer 1 at 1000K
Unexpected error for analyzer 1 at 1000K: VanHoveAnalysis.get_3d_plot() got an unexpected keyword argument 'ax'
Starting Van Hove Analysis for analyzer 2 at 1200K
Unexpected error for analyzer 2 at 1200K: VanHoveAnalysis.get_3d_plot() got an unexpected keyword argument 'ax'
Starting Van Hove Analysis for analyzer 3 at 1400K
Unexpected error for analyzer 3 at 1400K: VanHoveAnalysis.get_3d_plot() got an unexpected keyword argument 'ax'
Starting Van Hove Analysis for analyzer 4 at 1600K
Unexpected error for analyzer 4 at 1600K: VanHoveAnalysis.get_3d_plot() got an unexpected keyword argument 'ax'
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by shyuep** [comment]
Fixed.
